web/src/components/widgets/Sidebar.tsx

```diff
@@ -0,0 +1,43 @@
+import { reposStore } from '@/store/repoStore'
+import { NavigationMenu } from '@kobalte/core/navigation-menu'
+import { createSignal, For } from 'solid-js'
+
+const Sidebar = () => {
+  const [isOpen, setIsOpen] = createSignal(false)
+
+  const repos = reposStore.repos.map((repo) => repo.fullName)
```

Comment:
it must be in `createEffect`, otherwise it doesn't update when the list if changed